File: app/src/main/java/io/keyu/urekalite/view/DetailsFragment.java
package io.keyu.urekalite.view;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.ImageView;
import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;
import io.keyu.urekalite.R;

public class DetailsFragment extends Fragment {

    /**
     * Create a new DetailsFragment
     */
    public static DetailsFragment newInstance() {
        DetailsFragment fragment = new DetailsFragment();
        return fragment;
    }

    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        return inflater.inflate(R.layout.fragment_details, container, false);
    }

    @Override
    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);
//        ImageView image = view.findViewById(R.id.image);
//        image.setImageResource(R.drawable.science);
    }
}

File: app/src/main/java/io/keyu/urekalite/view/DetailsFragment.kt
package io.keyu.urekalite.view

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import io.keyu.urekalite.R

class DetailsFragment : Fragment() {

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        return inflater.inflate(R.layout.fragment_details, container, false)
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
        //        ImageView image = view.findViewById(R.id.image);
        //        image.setImageResource(R.drawable.science);
    }

    companion object {

        /**
         * Create a new DetailsFragment
         */
        fun newInstance(): DetailsFragment {
            return DetailsFragment()
        }
    }
}
